### DISCUSSION TITLE
Webinterface Redesign

### DISCUSSION THREAD

**Discussion by scfede** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#65] [Ideas]
I'm thinking about a redesign of the webinterface. Should be modern, clean and perhaps multi-language. Not pretty sure how many people using the webinterface vs. a smart-home integrated solution. Can someone give fedback on this?

I will do this in my own fork first but can provide the results as a pull-request to let you all benefit from this work.

Any ideas from you, the community on this task? What do you think about it?

**Comment by kiplin1995** [comment]
I'd be interested to see what you come up with.
I tweaked mine a little to be more to my taste but I inevitably ran out of will power and inspiration so left it here

![49658F8B-1785-47AC-8B8A-2C58640D80C9](https://user-images.githubusercontent.com/82231125/117570656-fe359300-b0c2-11eb-8d68-aeb17ebfa1b9.jpeg)

Currently only sat at 30°C as I’m away from home for a few days!
